cypress/e2e/index.cy.ts

```diff
@@ -178,6 +178,13 @@ describe("Case list", () => {
         cy.visit("/bichard")
 
         cy.findByText("Court Name").click()
+        cy.findByText("Court Name").parent().siblings().get('*[class^="icon"]').should("exist")
+        cy.findByText("Court Name")
+          .parent()
+          .siblings()
+          .get('*[class^="icon"]')
+          .get('*[class^="upArrow"]')
```

Comment:
since the `upArrow` and `downArrow` classes aren't generated by `react-jss` and are static, we can shorten this to `.upArrow` instead of `*[class^="upArrow"]`